packages/gatsby/src/commands/repl.js

```diff
@@ -0,0 +1,54 @@
+const repl = require(`repl`)
+const { graphql } = require(`graphql`)
+const bootstrap = require(`../bootstrap`)
+const {
+  store,
+  loadNodeContent,
+  getNodes,
+  getNode,
+  hasNodeChanged,
+} = require(`../redux`)
+
+module.exports = async program => {
+  // run bootstrap
+  await bootstrap(program)
+
+  // get all the goodies from the store
+  const {
+    schema,
+    config,
+    babelrc,
+    jsonDataPaths,
+    pages,
+    components,
+    staticQueryComponents,
+    nodes,
+  } = store.getState()
+
+  const query = async query => {
+    const result = await graphql(schema, query, {}, {}, {})
+    console.log(`query result: ${JSON.stringify(result)}`)
+  }
+
+  // init new repl
+  const _ = repl.start({
+    prompt: `gatsby > `,
+  })
+
+  // set some globals to make life easier
+  _.context.babelrc = babelrc
+  _.context.components = components
+  _.context.dataPaths = jsonDataPaths
+  _.context.getNode = getNode
+  _.context.getNodes = getNodes
+  _.context.hasNodeChanged = hasNodeChanged
+  _.context.loadNodeContent = loadNodeContent
+  _.context.nodes = [...nodes.entries()]
+  _.context.pages = [...pages.entries()]
+  _.context.runQuery = query
```

Comment:
Could we just name this "graphql" to match the function we pass around internally?